keepalived-vip/controller.go

```diff
@@ -284,12 +302,13 @@ func (ipvsc *ipvsControllerController) Stop() error {
 }
 
 // newIPVSController creates a new controller from the given config.
-func newIPVSController(kubeClient *unversioned.Client, namespace string, useUnicast bool, configMapName string, vrid int) *ipvsControllerController {
+func newIPVSController(kubeClient *unversioned.Client, namespace string, useUnicast bool, configMapName string, vrid int, dryrun bool) *ipvsControllerController {
```

Comment:
I was thinking on that. I thought better to have one flag than to have a common global shared object or flag for all other objects.
Only Idea I had was another method setDryRun(), but this Comes to late.

Do you have a better idea?